Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM pytorch/pytorch:2.5.1-cuda11.8-cudnn9-devel
+
+WORKDIR /workspace
+
+RUN apt-get update && \
+    apt-get install wget -y && \
+    apt-get install git -y
+
+# Install Miniconda
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh
```

Comment:
Would you maybe be able to consider using a free conda implementation? E.g. Miniforge from [conda-forge](https://conda-forge.org/download/)?

It's hard to predict what will happen to the permissiveness of Anaconda products: https://www.theregister.com/2024/08/08/anaconda_puts_the_squeeze_on/

If not, please add an echo or some sort of warning for users to check Anaconda's licensing terms.
Thanks!